lib/foreman_maintain.rb

```diff
@@ -57,5 +58,15 @@ def available_checks(*args)
     def available_procedures(*args)
       detector.available_procedures(*args)
     end
+
+    def init_logger
+      # Note - If timestamp added to filename then number of log files i.e second
+      # argument to Logger.new will not work as expected
+      filename = File.expand_path("#{config.log_dir}/fm.log")
```

Comment:
small note: `fm.log` is not very descriptive. Havnig `foreman-maintain.log` would be more suitable